data.sql

```diff
@@ -13,3 +13,26 @@ VALUES
   ('Boarmon', '2005-06-07', 7, true, 24.00),
   ('Blossom', '1998-10-13', 3, true, 17.00),
   ('Ditto', '2022-05-14', 4, true, 22.00);
+
+INSERT INTO owners (full_name, age)
+VALUES 
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 18);
```

Comment:
- [ ] Please add the same data given by microverse, for now, the `age` of `Jodie Whittaker` is `18` but in microverse provided data the `age` of `Jodie Whittaker` is `38` so kindly change this data and fix this issue as well :slightly_smiling_face: